### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by molleer**
Open for suggestions
This calendar is not done yet but the PR allows people to vent their opinions so far.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     "jss": "10.0.0",
     "lodash": "4.17.15",
     "prop-types": "15.7.2",
-    "react-markdown": "4.1.0",
     "react-is": "^16.10.2",
+    "react-markdown": "4.1.0",
```
Comment (Hoidi) [inline package.json]: why

Comment (molleer) [reply package.json]: Dont know. I was testing bootstrap before.

Comment (Hoidi) [reply package.json]: I was more commenting on that you removed `"react-markdown": "4.1.0",` and added it again 👍 Doesn't matter really. Just found it funny

**Comment by Portals** [comment]
Why is rollup config removed?

**Comment by molleer** [comment]
Is this component ready for production? @Portals @Hoidi

**Pull request closed without merging**